src/library/playlisttablemodel.cpp

```diff
@@ -36,20 +45,13 @@ void PlaylistTableModel::setTableModel(int playlistId) {
             // the same value as the cover hash.
             << LIBRARYTABLE_COVERART_HASH + " AS " + LIBRARYTABLE_COVERART;
 
-    // We drop files that have been explicitly deleted from mixxx
```

Comment:
Is this comment correct? Do we delete tracks from playlists when we hide them? I thought we only removed tracks from playlists on purge.